ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/compute_kernel_config.hpp

```diff
@@ -44,7 +46,7 @@ inline DeviceComputeKernelConfig init_device_compute_kernel_config(
                     defaultConfig = GrayskullComputeKernelConfig{
                         .math_fidelity = math_fidelity, .math_approx_mode = math_approx_mode};
                 } else if constexpr (std::is_same_v<T, WormholeComputeKernelConfig>) {
-                    TT_ASSERT(arch == ARCH::WORMHOLE_B0, "kernel config is not for wormhole_b0");
+                    TT_ASSERT(arch == ARCH::WORMHOLE_B0 or arch == ARCH::BLACKHOLE, "kernel config is not for wormhole_b0 or blackhole");
```

Comment:
Can you  please add a function called `is_wormhole_or_blackhole(arch)`?

And replace all of code like these to be `arch() == ARCH::WORMHOLE_B0 or arch() == ARCH::BLACKHOLE` use it

Reply:
done